src/gui/TotpExportSettingsDialog.ui

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>TotpExportSettingsDialog</class>
+ <widget class="QWidget" name="TotpExportSettingsDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>264</width>
+    <height>194</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Scan TOTP Settings</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <widget class="QLabel" name="totpKeyLabel">
+     <property name="font">
+      <font>
+       <pointsize>53</pointsize>
+      </font>
+     </property>
+     <property name="text">
+      <string></string>
+     </property>
+     <property name="alignment">
+      <set>Qt::AlignCenter</set>
+     </property>
+    </widget>
+   </item>
+   <item>
+     <layout class="QHBoxLayout" name="horizontalLayout">
+       <property name="alignment">
+        <set>Qt::AlignCenter</set>
+       </property>
+       <item>
+        <widget class="QCheckBox" name="enableAutoCloseCheckBox">
+         <property name="text">
+          <string></string>
```

Comment:
I don't like that this checkbox has two different functions in one and changes text based on checked status. It also moves the checkbox around when it's centered.

Reply:
Which two functions?

Reply:
It affects the current export window and sets a global configure option as well. Also the label should work to change the checked status.

I would prefer removing the auto-close feature completely as suggested before. It's not time critical to close the window and it removes the double function checkbox that moves around.

Reply:
> It affects the current export window and sets a global configure option as well.

With this, you mean that the option should be configurable per entry? I think it's preferable that the checkbox affects a global setting.

>I would prefer removing the auto-close feature completely as suggested before. It's not time critical to close the window and it removes the double function checkbox that moves around.

The primary motivation for the auto-close was to prevent blocking the auto-close feature since we're using a modal window. I agree it's ugly that the checkbox "moves around" when the text changes. Perhaps this can be fixed, instead of dropping it altogether?

Reply:
> With this, you mean that the option should be configurable per entry? I think it's preferable that the checkbox affects a global setting.

I mean that a global option shouldn't be on the dialog.

> Perhaps this can be fixed, instead of dropping it altogether?

Having the auto-close as a global option in the settings and on by default would possibly be a solution. Only show the time on the window then.

Reply:
@hifi What do you think of the latest changes addressing this issue? Here is a screenshot: 

![screenshot here](https://cdn.pbrd.co/images/H2Z5KS6.png)